rc-events-service/src/main/scala/com/epam/random_coffee/events/api/response/EventView.scala

```diff
@@ -0,0 +1,9 @@
+package com.epam.random_coffee.events.api.response
+
+import com.epam.random_coffee.events.model.{ Event, EventId }
+
+case class EventView(id: EventId, eventName: String)
```

Comment:
```suggestion
case class EventView(id: EventId, name: String)
```
[minor] since you're using just `name` in other places

Reply:
fixed